configure_cake_wallet.sh

```diff
@@ -27,9 +27,41 @@ if [ "$PLATFORM" == "$ANDROID" ]; then
     cd scripts/android
 fi
 
+echo "============================ BUILDING DEPS ============================"
+
 source ./app_env.sh cakewallet
 ./app_config.sh
+./build_decred.sh
```

Comment:
Build scripts aren't really supposed to be part of this configure script. It seems this script is primarily meant to configure the flutter build environment, because it does not include the other build (_monero, _haven, etc) scripts.